phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -952,14 +954,19 @@ private void addCoprocessors(byte[] tableName, TableDescriptorBuilder builder, P
                     if (newDesc.hasCoprocessor(PhoenixTransactionalIndexer.class.getName())) {
                         builder.removeCoprocessor(PhoenixTransactionalIndexer.class.getName());
                     }
-                    if (globalIndexerEnabled) {
-                        if (!newDesc.hasCoprocessor(IndexRegionObserver.class.getName())) {
+                    if (indexRegionObserverEnabled) {
+						if (newDesc.hasCoprocessor(Indexer.class.getName())) {
```

Comment:
Please fix the indentation. It got really messed up after latest forced push.

Reply:
Thanks for noticing. Didn't realize it in the local diff.